releases/release-1.13/release-notes-draft.md

```diff
@@ -1,278 +1,441 @@
 
-- Move scheduler cache interface and implementation to pkg/scheduler/internal/cache. ([#68968](https://github.com/kubernetes/kubernetes/pull/68968), [@wgliang](https://github.com/wgliang))
+#### aws-alb-ingress-controller v1.0.0
 
-### SIG API Machinery, and SIG Architecture
+- [Documentation](https://kubernetes-sigs.github.io/aws-alb-ingress-controller)
 
-- The caBundle and service fields in admission webhook API objects now correctly indicate they are optional ([#70138](https://github.com/kubernetes/kubernetes/pull/70138), [@liggitt](https://github.com/liggitt))
+##### Changelog since 1.0-beta.7
 
-### SIG Network, and SIG Windows
+###### Action Required
 
-- Fix service issues with named targetPort for Windows ([#70076](https://github.com/kubernetes/kubernetes/pull/70076), [@feiskyer](https://github.com/feiskyer))
+- change annotation `security-group-inbound-cidr` to `inbound-cidr`([#733](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/733), [@M00nF1sh](https://github.com/m00nf1sh))
 
-### SIG Azure, and SIG Storage
+###### Other notable changes
 
-- fix GetVolumeLimits log flushing issue ([#69558](https://github.com/kubernetes/kubernetes/pull/69558), [@andyzhangx](https://github.com/andyzhangx))
+* Added support for specify default tags at controller level([#711](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/711), [@jmcarp](https://github.com/jmcarp))
+* Added feature gate for WAF support([#728](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/728), [@M00nF1sh](https://github.com/m00nf1sh))
+* Allow service of type LoadBalancer for instance target mode([#732](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/732), [@hatmatter](https://github.com/hatmatter))
+* Align tagging support for securityGroups with other resources([#730](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/730), [@M00nF1sh](https://github.com/m00nf1sh))
+* Fixed bug when handling empty path in ingress spec([#736](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/736), [@M00nF1sh](https://github.com/m00nf1sh))
+* Resolve VPC ID and AWS Region from ec2metadata during start up([#723](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/723), [@M00nF1sh](https://github.com/m00nf1sh))
 
-### SIG Apps, SIG Scheduling, and SIG Testing
 
-- Scheduling conformance tests related to daemonsets should set the annotation that relaxes node selection restrictions, if any are set. This ensures conformance tests can run on a wider array of clusters. ([#68793](https://github.com/kubernetes/kubernetes/pull/68793), [@aveshagarwal](https://github.com/aveshagarwal))
+#### aws-ebs-csi-driver v0.1.0-alpha.1
```

Comment:
All of these are changes to an external CSI driver `https://github.com/kubernetes-sigs/aws-ebs-csi-driver`. They are not part of the Kubernetes 1.13 release.

Please remove all references to `aws-ebs-csi-driver`.

Reply:
Adding in @d-nishi and @AishSundar as we had a chat about whether or not this was appropriate for the release notes. I'll note that there was a similar conversation on notes for kubeadm vs kops/kubespray/etc and it was asserted that since `kubeadm` is apart of the release artifacts for a Kubernetes release, it's appropriate to include them in the notes.

Reply:
CSI drivers are currently not shipped as part of Kubernetes. The release cadence of external CSI drivers doesn't necessarily align with Kubernetes releases. So it really makes no sense to include release notes of a CSI driver here at this point.

The release notes already link to https://kubernetes-csi.github.io/docs/Drivers.html where folks can find a list of drivers.

Reply:
@saad-ali -- this is not just about CSI driver. It is about all sub-projects that sit in k8s-sigs and are out-of-tree but work with k8s. We discussed this at the very beginning of the release cycle. Sub-projects under k8s-sigs though out of tree, benefit from being tagged to a specific k8s release because we add CI signal before moving the project from alpha, beta to GA. This discipline makes it possible for users to know which k8s version to test the driver with atleast in AWS. we pegged the aws-ebs-csi-driver to k8s 1.13 for all testing.

Reply:
Adding @justaugustus to lend some color to the discussion we had around release notes mention for out-of-tree projects.

Reply:
@saad-ali Your logic holds good for CSI drivers but not for other sub-projects related to providers. I also don't believe all 17 drivers have some commentary available every cycle.

Albeit, I believe your main point is long release notes increase the cognitive burden on the user. 

Therefore, suggestions: a) We can create a separate section in the release notes for out-of-tree projects which gives a user the choice to ignore whats not essential but does not get rid of valid information; or 
b) We can link k8s-sigs release notes under SIG-AWS with a summary as follows:
https://github.com/kubernetes-sigs/aws-alb-ingress-controller/releases/tag/v1.0.0 
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/CHANGELOG-0.1.md
[prefer option b. given the idea is to reduce cognitive load for the user]

The lack of either option: 
a) makes it very hard for our users to know what really is going on; 
b) As SIG-AWS lead we cannot communicate what value we deliver every release cycle neither to users nor to the community.

Reply:
We obviously can't relnote "all possible external components".  That isn't the debate.  This is part of kubernetes-sigs, so it is kind of part of the project.

My feeling is that things that are not part of the RELEASE ARTIFACT(S) probably should not be in the release notes.  My belief is that vendor-driven drivers should live in vendor-specific places.  The fact that this one lives in kubernetes-sigs  is either a historical artifact, or a legal issue (changing CLAs retroactively), or a community-written driver for an absentee vendor (doesn't seem to be the case here).

What I want to prevent is creating an incentive for vendors to dump stuff in kubernetes-sigs, when they don't HAVE TO.

I'd like to hear the release-team on this, but that's my lean.

Reply:
@thockin -- some more data for reference here and happy to go with recommendations. 

We followed process (CLA, license, ownership by a SIG, KEP, lazy consensus and CI signal integration with a specific k8s version in mind) when defining subprojects and tagged them to SIG-AWS before hosting under k8s-sigs as documented here: https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md

Reply:
So, I just skimmed the relnotes and the cognitive burden is real.

I agree with what you're all saying...
- artifacts that are not created as part of this release process, should not appear in these relnotes
- it's [impossible|not feasible|difficult] to enumerate all external components
- some external components are dependent on artifacts generated from this process and should denote supportability in the respective repos, not the other way around

w.r.t. @d-nishi, yes, she'd followed every documented process to the letter, and has been diligent in reaching out to people to clarify things when there have been gaps. If we have feelings about vendor-specific things, we need to make them clear in the k/org processes and not here.

We need to be able to recognize the works of SIGs, but the relnotes are not the place.

I like the idea of linking out to SIG specific notes for a more detailed overview of what's been happening and I think in future that this would happen in the context of the Contributor Site.

Right now, we need to remove mention of anything that isn't release within k/k and link to their relnotes instead.

Reply:
To be clear, I'm voting option B here.

Reply:
This was bit of a new territory for the release team as well since there was no clear precedent of how much of the k8s release process (feature graduation stages) and relnote should apply to out-of-tree features developed as part of kubernetes-sigs. Technically this does not produce any 1.13 release artifact to go into the relnotes. That said, since we dont have any clear guideline around this, I look to SIG PM, and @justaugustus who was driving the effort outline guidelines for out-of-tree features. Personally I think option (b) mentioned above my Nishi is a happy medium until we can have clear guidelines to apply to all such cases in the future.

Reply:
Agreed that we won't have a good answer for this until we have guidelines. I'll work to drive that down in 1.14.

Reply:
> Albeit, I believe your main point is long release notes increase the cognitive burden on the user.

Yes!

> My belief is that vendor-driven drivers should live in vendor-specific places. The fact that this one lives in kubernetes-sigs is either a historical artifact, or a legal issue (changing CLAs retroactively), or a community-written driver for an absentee vendor (doesn't seem to be the case here).

Yes!

Option b is a better option then option a. I like the idea of cloud providers having a page on their repo with a collection of release notes for their components.

However, even linking to these pages from the general kubernetes release notes seems wrong to me: it gives a set of vendors (i.e. cloud providers) a way to promote external components (that may not ship with Kubernetes) while vendors that don't have a SIG don't get to do the same.

To me release notes are here is what is "in the box".

I agree the problem of discoverability of external-components (and changes to them) is real. I don't have a great answer for how to fix that. But the core Kubernetes release notes seems like the wrong answer.

Reply:
Hey y'all (@d-nishi, @saad-ali, @AishSundar, et al), I took a run at adding a paragraph to SIG AWS's Major Themes which links to the documentation for each of these external projects: https://github.com/kubernetes/sig-release/pull/386/commits/997ba6a19041c45a6698be4e8ce78542af70d228.

What do y'all think about this? IMO it definitely makes the notes a lot easier to parse as there isn't a big block of sections with 6 levels of headers.

Reply:
@marpaia - sent you an email with some wording change but linked to release notes of the subprojects. please use the same - Cheers!

Reply:
resolved this with @marpaia with suggested Option (b) aka linked to subproject changelogs and reduced the description to one paragraph.

Reply:
Thanks all, I know this is new territory.  We will have more of this as the project components get looser coupling.  I didn't mean to imply any procedural mistakes or ill intent.